pkg/domain/entities/containers.go

```diff
@@ -164,6 +164,7 @@ type ContainerStatReport struct {
 type CommitOptions struct {
 	Author         string
 	Changes        []string
+	Config         io.Reader
```

Comment:
This struct is used for the REST API as well - will this break given that io.Reader can't really be serialized over the network?

Reply:
Changed it back to a `[]byte`.